python/dgl/utils.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=C0103
```

Comment:
What is this?

Reply:
Invalid constant name. Related to the function such as STR2DTYPE, will refactor those